glide.yaml

```diff
@@ -18,10 +19,10 @@ import:
 
 # crd-schema-gen
 - package: sigs.k8s.io/controller-tools
-  version: 43466124052c1a1aa7d8fd33624b00bc111fb7cf
-  repo: https://github.com/openshift/kubernetes-sigs-controller-tools.git
+  repo: https://github.com/openshift/kubernetes-sigs-controller-tools
+  version: master
```

Comment:
this looks unlikely to keep working well.  If this is a fork, you need a branch.  If we really want to track a master, pin to sha.

Reply:
https://github.com/sttts/controller-tools/tree/master-1.13-rebase – how should we call it?

Reply:
Tracking branch now with standard name.